docs/modules/SDDs/pages/getting-started.adoc

```diff
@@ -0,0 +1,168 @@
+= Getting Started with Project Syn
+
+This guide helps to get started with the various Project Syn tools.
+
+== Requirements
+
+* Access to a Kubernetes cluster. You can also use a local cluster, f.e. https://k3s.io/[k3s] with https://k3d.io/[k3d] works perfectly fine.
+* A GitLab.com Account
+* `curl`
+
+== Kickstart Lieutenant
+
+Create a namespace to host Lieutenant Operator and API
+[source,shell]
+----
+kubectl create namespace lieutenant
+----
+
+=== Install Lieutenant Operator
+
+Install *Lieutenant Operator* with the following commands:
+
+[source,shell]
+----
+# CRDs
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_clusters_crd.yaml
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_gitrepos_crd.yaml
+kubectl -n lieutenant apply -f https://raw.githubusercontent.com/projectsyn/lieutenant-operator/master/deploy/crds/syn.tools_tenants_crd.yaml
+
+# Deployment
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/service_account.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/role.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/role_binding.yaml
+kubectl -n lieutenant apply -f https://github.com/projectsyn/lieutenant-operator/raw/master/deploy/operator.yaml
+
+# Patch version
```

Comment:
Should we add a command to add the `SKIP_VAULT_SETUP=true` env var?

Reply:
Yes, that's already in my local document, coming soon to this PR